### PR TITLE
Word counter #135

### PR DESCRIPTION
**Pull request by kshitija-2001**
### Description

Word counter app 

### Checklist

- [x] I've been assigned an issue related to this PR.
- [ ] I've used beautifiers.
- [x] I've added my Project's name and description to `Index.md`
- [x] I've made a README.md file for my Project.
- [ ] The README.md file of my project contains Project title, Description, Use of project, Set up, Stack used and Output (Screenshots). 

## Related Issues or Pull Requests number

(Write your answer here.)

### CODE REVIEW

File: word counter/readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#### Steps to Use:
+
+---
+
+- Download or clone the repository
+
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
```
Comment (harshita2216) [inline word counter/readme.md:25]: @kshitija-2001, please elaborate the steps.

Comment (harshita2216) [inline word counter/readme.md:25]: @kshitija-2001, please elaborate the steps.

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Add your project to index.md file as well.

**Comment by kshitija-2001** [comment]
corrected the requested changes

File: word counter/readme.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Go to the directory
+- Run the index.html file
+- Start counting Words!!
+
```
Comment (harshita2216) [inline word counter/readme.md:29]: Screenshots? @kshitija-2001

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Everything else looks fine!

File: word counter/readme.md (hunk 2 above)
Comment (kshitija-2001) [reply word counter/readme.md]: done added one screenshot

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**